docs/HowToFAQ/FAQNewline.md

```diff
@@ -8,6 +8,21 @@ It looks like, when compiling to C#, my print statements don't show up if I don'
 
 ## Answer
 
-This is likely to be platform-dependent problem; the problem is whether the I/O system for the target platform does a flush of the output streams prior to terminating a program. The workaround is for the user program to do a `print "\n";` at the end of the program.
+The desired output is present in all current Dafny backends. But note that
+if the print statement does not include the \n, then the output may not either.
+In that case the output may be (depending on the shell you are using)
+concatenated with the prompt for the next user input.
```

Comment:
Does pandoc remove these line breaks?  I think so but I know github doesn't always)

Reply:
Not sure which line break you mean? (I did change \n to newline)

Reply:
This paragraph is hard-wrapped, and I was curious whether pandoc would keep the hard wraps.  I think `\n` was good, since that's what the Dafny program text should contain.

Reply:
The hard wraps are not a problem for markdown nor for pandoc's interpretation of markdown.

Reply:
Sweet, thanks (they are a problem for Github flavored markdown, but IIRC only in certain places)